CI-CD/Jenkinsfile.Mable

```diff
@@ -0,0 +1,67 @@
+pipeline {
+	agent any
+	
+	tools { nodejs "node" }
+	
+	environment {
+		registryCredential = 'ecr:us-east-1:awscreds'
+		appRegistry = "947036392937.dkr.ecr.us-east-1.amazonaws.com/comosus-client-img"
+		comosusClientRegistry = "https://947036392937.dkr.ecr.us-east-1.amazonaws.com"
+		cluster = "ComosusClient"
+        service = "ComosusClientAppSVC"
+	}
+
+	stages {	
+		stage('Clear space'){
+		  steps {
+			    echo 'Cleanup docker space'
+                sh 'docker system prune -f'
+		  }
+		}
+		
+		stage('Fetch code'){
+		  steps {
+			git branch: 'develop', url: 'https://github.com/mable-pmyip/comosus-client.git'
+		  }
+		}
+		
+		stage('Install and Build'){
+		  steps {
+		    sh 'yarn install && yarn codegen && yarn build'
+		  }
+		}
+
+		stage('Test'){
+		  steps {
+			sh 'echo unit testing'
+		  }
+		}
+		
+		stage('Build App Image') {
+		   steps {
+            script {
+				dockerImage = docker.build( appRegistry + ":$BUILD_NUMBER", "-f Dockerfile.Mable .")
+			}
+		 }
+		}
+
+		stage('Upload App Image') {
+		  steps{
+			script {
+			  docker.withRegistry(comosusClientRegistry, registryCredential ) {
+				dockerImage.push("$BUILD_NUMBER")
+				dockerImage.push('latest')
```

Comment:
why do you need to push the image twice

Reply:
I copied from somewhere but I think it is a safety approach, in case we deleted the latest image, we still can access the image by the build number. 
![image](https://user-images.githubusercontent.com/94888732/174790965-7f34a0ca-b7cc-4124-94d8-8a7a0f7bdf57.png)

Reply:
I would recommend you to add some comments to explain the logics.